integrations/airflow/setup.py

```diff
@@ -21,7 +21,7 @@
 
 requirements = [
     "attrs==19.3",
-    "requests==2.25.1",
+    "requests>=2.25.1",
```

Comment:
I think here we need to adjust it to a lower version.

Reply:
Yeah, I've lowered it to `2.24.0`